graphbolt/src/cuda/index_select_impl.cu

```diff
@@ -160,14 +164,37 @@ torch::Tensor UVAIndexSelectImpl_(torch::Tensor input, torch::Tensor index) {
  * The supporting index types are: int, int64_t.
```

Comment:
Do we need to update the supporting types?

Reply:
However, index types are int32_t and int64_t still.